app/lnd/methods/networkController.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * [info description]
+ * @param  {[type]} lnd [description]
+ * @return {[type]}     [description]
+ */
+export function getInfo(lnd) {
+  return new Promise((resolve, reject) => {
+
+    lnd.getInfo({}, (err, data) => {
+      if (err) { reject(err) }
+
+      resolve(data)
+    })
+  })
+}
+
+
+/**
+ * Returns general information concerning the lightning node
+ * @param  {[type]} lnd    [description]
+ * @param  {[type]} pubkey [description]
+ * @return {[type]}        [description]
+ */
+export function getNodeInfo(lnd, { pubkey }) {
+  return new Promise ((resolve, reject) => {
+
+    lnd.getNodeInfo({ pub_key: pubkey}, (err, data) => {
+      if(err) { reject (err) }
+
+      resolve(data)
+    })
+  })
+}
+
+
+/**
+ * Returns a description of the latest graph state from the point of view of the node
+ * @param  {[type]} lnd [description]
+ * @return {[type]}     [description]
+ */
+export function describeGraph(lnd) {
+  return new Promise ((resolve, reject) => {
```

Comment:
Some small inconsistencies with space before `((resolve, reject) => ...`

I don't have a preference but let's pick no space or space and keep a standard

Reply:
`prettier` would kill the space! https://www.npmjs.com/package/prettier